deploy/crds/kieapp.crd.yaml

```diff
@@ -1913,6 +1913,35 @@ spec:
                                 type: string
                               type: array
                           type: object
+                        kafkaJbpmEventEmitters:
+                          description: KafkaJBPMEventEmittersObject kafka configuration
+                            to be used by the KieApp for JBPM Emitter
+                          properties:
+                            acks:
+                              description: Jbpm Emitter, the number of acknowledgments
```

Comment:
please, align the descriptions here with the suggestions I made on the rhpam image.yaml, does not need to add the final part where it mentions what system property it sets.

Reply:
Done

Reply:
didn't see the changes here, the description here https://github.com/jboss-container-images/rhpam-7-openshift-image/pull/550/files is not correct, but here still not updated.

Reply:
updated